tests/tools/fswalker/walker/walker.go

```diff
@@ -65,5 +65,6 @@ func WalkPathHash(ctx context.Context, path string) (*fspb.Walk, error) {
 		Include:         []string{path},
 		HashPfx:         []string{""}, // Hash everything
 		MaxHashFileSize: MaxFileSizeToHash,
+		WalkCrossDevice: true,
```

Comment:
This permits the walk to cross device (or probably mount-point) boundaries.
Was the error during the walk or when comparing the reports?

Reply:
From the error message, it appears to be during the walk

Reply:
Was during the walk, here:
https://github.com/google/fswalker/blob/7aa03fed36caf47cb8b384ef400778ef73f09db6/walker.go#L289